gui/services/migrations/0021_add_enable_smb1.py

```diff
@@ -0,0 +1,42 @@
+from django.db import migrations, models
```

Comment:
This migration needs to remain the same as 11.1-stable. Same name, same dependency. What you will need is a merge migration

manage.py makemigrations services --merge

Reply:
Okay. Thanks. I was not familiar with the process.

Reply:
We have 0020 and 0021 being the same, 0021 should probably be removed.

Reply:
Yes. That was an accident.